frontend/src/app/app/sidebar.tsx

```diff
@@ -3,7 +3,7 @@ import Link from "next/link";
 
```

Comment:
why are you creating a new sidebar file when theres already one used in the /app/profile ?

just import that one and use it man.

Also theres a component folder in the /app/quests folder, move your ranking table in there.

Reply:
I used the same side nav used in profile. I just changed the text color to white as instructed by @DanielEmmanuel1

Reply:
@Dprof-in-tech though i noticed there was another sidebar components located in the app/quests/socials/components which is still in works